<!DOCTYPE html>
<html lang="en-us">

  <head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="description" content="Your description">
  <meta name="author" content="Your name">
  <meta name="generator" content="Hugo 0.19" />
  <title>BBSRC Bistability of Cell Cycle Transitions Project</title>
  <!-- Stylesheets -->
  <link href="https://novakgroupoxford.github.io/bicycle-hugo/css/bootstrap-v3.3.7/bootstrap.min.css" rel="stylesheet">
  
  <link href="http://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
  
  <link type="text/css" rel="stylesheet" href="https://novakgroupoxford.github.io/bicycle-hugo/css/materialize.css"  media="screen,projection"/>
  <link href="https://novakgroupoxford.github.io/bicycle-hugo/css/agency.css" rel="stylesheet">


  

  <!-- Custom Fonts -->
  <link href="https://novakgroupoxford.github.io/bicycle-hugo/font-awesome-v4.7.0/css/font-awesome.min.css" rel="stylesheet" type="text/css">
  <link rel="stylesheet" href="https://novakgroupoxford.github.io/bicycle-hugo/academicons-1.7.0/css/academicons.min.css"/>


  
    <link href="https://fonts.googleapis.com/css?family=Montserrat:400,700" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Kaushan+Script" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Droid+Serif:400,700,400italic,700italic" rel="stylesheet" type="text/css">
    <link href="https://fonts.googleapis.com/css?family=Roboto+Slab:400,100,300,700" rel="stylesheet" type="text/css">
  

  <!-- HTML5 Shim and Respond.js IE8 support of HTML5 elements and media queries -->
  <!-- WARNING: Respond.js doesn't work if you view the page via file:// -->
  <!--[if lt IE 9]>
  <script src="https://oss.maxcdn.com/libs/html5shiv/3.7.0/html5shiv.js"></script>
  <script src="https://oss.maxcdn.com/libs/respond.js/1.4.2/respond.min.js"></script>
  <![endif]-->
</head>


  <body id="page-top" class="index">
    <!-- Navigation -->
<nav class="navbar navbar-default navbar-fixed-top">
  <div class="container">

    <!-- Brand and toggle get grouped for better mobile display -->
    <div class="navbar-header page-scroll">
      <button type="button" class="navbar-toggle" data-toggle="collapse" data-target="#bs-example-navbar-collapse-1">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      
        <a class="navbar-brand page-scroll" href="https://novakgroupoxford.github.io/bicycle-hugo/">
          <img src="https://novakgroupoxford.github.io/bicycle-hugo//img/bi-cycle.png" class="img-responsive" alt="bicycle">
        </a>
      

    </div>
    <!-- Collect the nav links, forms, and other content for toggling -->
    <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
      <ul class="nav navbar-nav navbar-right">
        <li class="hidden">
          <a href="https://novakgroupoxford.github.io/bicycle-hugo/#page-top"></a>
        </li>

        

        
        <li>
          <a  href="https://novakgroupoxford.github.io/bicycle-hugo/#about">About</a>
        </li>
        

        <li>
          <a  href="https://novakgroupoxford.github.io/bicycle-hugo/news">News</a>
        </li>

        

        



        
        <li>
          <a href="https://novakgroupoxford.github.io/bicycle-hugo/member">Members</a>
        </li>
        <li>
          <a href="https://novakgroupoxford.github.io/bicycle-hugo/publication">Publications</a>
        </li>

        
          <li>
            <a href="https://novakgroupoxford.github.io/bicycle-hugo/static/PPSIM/timecourse.html">PPSIM</a>
          </li>
        
      </ul>
    </div>
    <!-- /.navbar-collapse -->
  </div>
  <!-- /.container-fluid -->
</nav>


    <div class="portfolio-modal fade in" id="portfolioModal_bela_novak" tabindex="-1" role="dialog" aria-hidden="true">
      <div class="modal-content">
        <div class="container">
          <div class="row">
            <div class="col-lg-8 col-lg-offset-2">
              <div class="modal-body">
                
                
                <h2>Bela Novak</h2>
                <h6 class="item-intro text-muted">Principal Investigator</h6>
                <img class="img-responsive img-centered img-circle" src="https://novakgroupoxford.github.io/bicycle-hugo/img/member/bela_novak.jpg" alt="">
                <p>Bela Novak trained as a biochemical engineer and started his research career as an experimental cell biologist working on the fission yeast cell cycle with Prof. Murdoch Mitchison. In the early nineties, in collaboration with Prof. John Tyson, he pioneered a systems-level description of eukaryotic cell cycle controls. The mathematical modelling approach that he established, based on biochemical reaction kinetics, computer simulations and dynamical systems theory, has proven effective in bridging the gap between molecular control networks and cell physiology. In his hands, these models proved successful in uncovering basic principles and molecular mechanisms of cell cycle control. Being well-versed in the experimental literature, he easily communicates with experimentalists, evidenced by his joint Cell and Nature publications with Dr Frank Uhlmann (CRUK, LRI) and Dr Wolfgang Zachariae (MPI, Martinsried) on the budding yeast cell cycle and meiosis. Recently, he has started successful collaborations with four experimentalists participating in this proposal (Prof. Francis Barr, Dr Helfrid Hochegger, Dr Ulrike Gruneberg and Dr Chris Bakal), which also led to publications in Molecular Cell and PloS Genetics</p>
                <ul class="list-inline social-buttons">
                  
                    
                  
                    
                    
                      <li><a href="https://scholar.google.com/citations?user=bRE-FaUAAAAJ&amp;hl=en&amp;oi=ao" target="_blank"><i class="ai ai-google-scholar"></i></a></li>
                    
                  
                    
                  
                    
                    
                      <li><a href="https://www.researchgate.net/profile/Bela_Novak" target="_blank"><i class="ai ai-researchgate"></i></a></li>
                    
                  
                    
                  
                    
                  
                </ul>
              </div>
            </div>
          </div>
        </div>
        <div class="container" id="publications">
          <div class="row">
            <div class="col-lg-12 text-center">
              <h4 class="section-heading">Publications contributed to in the context of this project</h4>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-8 col-lg-offset-2">
              <ul class="timeline">
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/barr_naturecommunications_2017.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Feb 1, 2017</h6>
      <h5 class="subheading">Nature Communications</h5>
      <a href="/bicycle-hugo/publication/barr_naturecommunications_2017/"><h4>DNA damage during S-phase mediates the proliferation-quiescence decision in the subsequent G1 via p21 expression (Under Revision)</h4></a>
      <h5 class="subheading">
        Alexis Barr, Samuel Cooper, Stefan Heldt, Francesca Butera, Henriette Stoy, Jörg Mansfeld, Bela Novak, Chris Bakal
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/" target="_blank" title="Visit publisher website">https://dx.doi.org/</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2017_Barr_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2017_Barr_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Following DNA damage caused by exogenous sources, such as ionising radiation, the tumour suppressor p53 mediates cell cycle arrest via expression of the CDK inhibitor, p21. However, the role of p21 in maintaining genomic stability in the absence of exogenous DNA damaging agents is unclear. Here, using live single cell measurements of p21 protein in proliferating cultures, we show that naturally-occurring DNA damage incurred over S-phase causes p53-dependent accumulation of p21 during mother G2- and daughter G1-phases. High p21 levels mediate G1 arrest via CDK inhibition, yet lower levels have no impact on G1 progression, and the ubiquitin ligases CRL4Cdt2 and SCFSkp2 couple to degrade p21 prior to the G1/S transition. Mathematical modelling reveals that a bistable switch, created by CRL4Cdt2, promotes irreversible S-phase entry by keeping p21 levels low, preventing premature S-phase exit upon DNA damage. Thus, we characterise how p21 regulates the proliferation-quiescence decision to maintain genomic stability.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/mochida_currbiol_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Nov 23, 2016</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/bicycle-hugo/publication/mochida_currbiol_2016/"><h4>Two Bistable Switches Govern M Phase Entry</h4></a>
      <h5 class="subheading">
        Satoru Mochida, Scott Rata, Hirotsugu Hino, Takeharu Nagai, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2016.10.022" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2016.10.022</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1016%2fj.cub.2016.10.022" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The abrupt and irreversible transition from interphase to M phase is essential to separate DNA replication from chromosome segregation. This transition requires the switch-like phosphorylation of hundreds of proteins by the cyclin-dependent kinase 1 (Cdk1):cyclin B (CycB) complex. Previous studies have ascribed these switch-like phosphorylations to the auto-activation of Cdk1:CycB through the removal of inhibitory phosphorylations on Cdk1-Tyr15 [1 and 2]. The positive feedback in Cdk1 activation creates a bistable switch that makes mitotic commitment irreversible [2, 3 and 4]. Here, we surprisingly find that Cdk1 auto-activation is dispensable for irreversible, switch-like mitotic entry due to a second mechanism, whereby Cdk1:CycB inhibits its counteracting phosphatase (PP2A:B55). We show that the PP2A:B55-inhibiting Greatwall (Gwl)-endosulfine (ENSA) pathway is both necessary and sufficient for switch-like phosphorylations of mitotic substrates. Using purified components of the Gwl-ENSA pathway in a reconstituted system, we found a sharp Cdk1 threshold for phosphorylation of a luminescent mitotic substrate. The Cdk1 threshold to induce mitotic phosphorylation is distinctly higher than the Cdk1 threshold required to maintain these phosphorylations—evidence for bistability. A combination of mathematical modeling and biochemical reconstitution show that the bistable behavior of the Gwl-ENSA pathway emerges from its mutual antagonism with PP2A:B55. Our results demonstrate that two interlinked bistable mechanisms provide a robust solution for irreversible and switch-like mitotic entry.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/cundell_jcb_2016.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Aug 22, 2016</h6>
      <h5 class="subheading">Journal of Cell Biology</h5>
      <a href="/bicycle-hugo/publication/cundell_jcb_2016/"><h4>A PP2A-B55 recognition signal controls substrate dephosphorylation kinetics during mitotic exit</h4></a>
      <h5 class="subheading">
        Michael Cundell, Lukas Hutter, Ricardo Nunes-Bastos, Elena Poser, James Holder, Shabaz Mohammed, Bela Novak, Francis Barr
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1083/jcb.201606033" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1083/jcb.201606033</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1083%2fjcb.201606033" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2016_Cundell_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2016_Cundell_et_al</a>
              </div>
            </div>
          </li>
        
        
          <li>
            <div class="collapsible-header">
              <i class="fa fa-area-chart"></i>
              <a href="https://novakgroupoxford.github.io/bicycle-hugo/static/PPSIM/timecourse.html" target="_blank" title="Visualisation Resource">View Datavisualisation</a>
            </div>
          </li>
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>PP2A-B55 is one of the major phosphatases regulating cell division. Despite its importance for temporal control during mitotic exit, how B55 substrates are recognized and differentially dephosphorylated is unclear. Using phosphoproteomics combined with kinetic modeling to extract B55-dependent rate constants, we have systematically identified B55 substrates and assigned their temporal order in mitotic exit. These substrates share a bipartite polybasic recognition determinant (BPR) flanking a Cdk1 phosphorylation site. Experiments and modeling show that dephosphorylation rate is encoded into B55 substrates, including its inhibitor ENSA, by cooperative action of basic residues within the BPR. A complementary acidic surface on B55 decodes this signal, supporting a cooperative electrostatic mechanism for substrate selection. A further level of specificity is encoded into B55 substrates because B55 displays selectivity for phosphothreonine. These simple biochemical properties, combined with feedback control of B55 activity by the phosphoserine-containing substrate/inhibitor ENSA, can help explain the temporal sequence of events during exit from mitosis.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
                  
                  
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/chica_currbiol_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Monday, Feb 8, 2016</h6>
      <h5 class="subheading">Current Biology</h5>
      <a href="/bicycle-hugo/publication/chica_currbiol_2016/"><h4>Nutritional Control of Cell Size by the Greatwall-Endosulfine-PP2A·B55 Pathway</h4></a>
      <h5 class="subheading">
        Nathalia Chica, Ana Rozalen, Livia Perez-Hidalgo, Angela Rubio, Bela Novak, Sergio Moreno
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cub.2015.12.035" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cub.2015.12.035</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1016%2fj.cub.2015.12.035" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Proliferating cells adjust their cell size depending on the nutritional environment. Cells are large in rich media and small in poor media. This physiological response has been demonstrated in both unicellular and multicellular organisms. Here we show that the greatwall-endosulfine (Ppk18-Igo1 in fission yeast) pathway couples the nutritional environment to the cell-cycle machinery by regulating the activity of PP2A·B55. In the presence of nutrients, greatwall (Ppk18) protein kinase is inhibited by TORC1 and PP2A·B55 is active. High levels of PP2A·B55 prevent the activation of mitotic Cdk1·Cyclin B, and cells increase in size in G2 before they undergo mitosis. When nutrients are limiting, TORC1 activity falls off, and the activation of greatwall (Ppk18) leads to the phosphorylation of endosulfine (Igo1) and inhibition of PP2A·B55, which in turn allows full activation of Cdk1·CyclinB and entry into mitosis with a smaller cell size. Given the conservation of this pathway, it is reasonable to assume that this mechanism operates in higher eukaryotes, as well.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/barr_cellsystems_2016.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jan 27, 2016</h6>
      <h5 class="subheading">Cell Systems</h5>
      <a href="/bicycle-hugo/publication/barr_cellsystems_2016/"><h4>A Dynamical Framework for the All-or-None G1/S Transition</h4></a>
      <h5 class="subheading">
        Alexis Barr, Stefan Heldt, Tongli Zhang, Chris Bakal, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.cels.2016.01.001" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.cels.2016.01.001</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1016%2fj.cels.2016.01.001" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>The transition from G1 into DNA replication (S phase) is an emergent behavior resulting from dynamic and complex interactions between cyclin-dependent kinases (Cdks), Cdk inhibitors (CKIs), and the anaphase-promoting complex/cyclosome (APC/C). Understanding the cellular decision to commit to S phase requires a quantitative description of these interactions. We apply quantitative imaging of single human cells to track the expression of G1/S regulators and use these data to parametrize a stochastic mathematical model of the G1/S transition. We show that a rapid, proteolytic, double-negative feedback loop between Cdk2:Cyclin and the Cdk inhibitor p27Kip1 drives a switch-like entry into S phase. Furthermore, our model predicts that increasing Emi1 levels throughout S phase are critical in maintaining irreversibility of the G1/S transition, which we validate using Emi1 knockdown and live imaging of G1/S reporters. This work provides insight into the general design principles of the signaling networks governing the temporally abrupt transitions between cell-cycle phases.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/mirkovic_cellreports_2015.jpg" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Oct 8, 2015</h6>
      <h5 class="subheading">Cell Reports</h5>
      <a href="/bicycle-hugo/publication/mirkovic_cellreports_2015/"><h4>Premature Sister Chromatid Separation Is Poorly Detected by the Spindle Assembly Checkpoint as a Result of System-Level Feedback</h4></a>
      <h5 class="subheading">
        Mihailo Mirkovic, Lukas Hutter, Bela Novak, Raquel Oliveira
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.celrep.2015.09.020" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.celrep.2015.09.020</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1016%2fj.celrep.2015.09.020" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
          <li>
            <div>
              <div class="collapsible-header">
                <i class="fa fa-github"></i>
                <a href="https://github.com/novakgroupoxford/2015_Mirkovic_et_al" target="_blank" title="View Code Resource on Github">https://github.com/novakgroupoxford/2015_Mirkovic_et_al</a>
              </div>
            </div>
          </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>Sister chromatid cohesion, mediated by the cohesin complex, is essential for faithful mitosis. Nevertheless, evidence suggests that the surveillance mechanism that governs mitotic fidelity, the spindle assembly checkpoint (SAC), is not robust enough to halt cell division when cohesion loss occurs prematurely. The mechanism behind this poor response is not properly understood. Using developing Drosophila brains, we show that full sister chromatid separation elicits a weak checkpoint response resulting in abnormal mitotic exit after a short delay. Quantitative live-cell imaging approaches combined with mathematical modeling indicate that weak SAC activation upon cohesion loss is caused by weak signal generation. This is further attenuated by several feedback loops in the mitotic signaling network. We propose that multiple feedback loops involving cyclin-dependent kinase 1 (Cdk1) gradually impair error-correction efficiency and accelerate mitotic exit upon premature loss of cohesion. Our findings explain how cohesion defects may escape SAC surveillance.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/tyson_bmcbiology_2015.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Wednesday, Jul 1, 2015</h6>
      <h5 class="subheading">BMC Biology</h5>
      <a href="/bicycle-hugo/publication/tyson_bmcbiology_2015/"><h4>Models in biology: lessons from modeling regulation of the eukaryotic cell cycle</h4></a>
      <h5 class="subheading">
        John Tyson, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1186/s12915-015-0158-9" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1186/s12915-015-0158-9</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1186%2fs12915-015-0158-9" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>In this essay we illustrate some general principles of mathematical modeling in biology by our experiences in studying the molecular regulatory network underlying eukaryotic cell division. We discuss how and why the models moved from simple, parsimonious cartoons to more complex, detailed mechanisms with many kinetic parameters. We describe how the mature models made surprising and informative predictions about the control system that were later confirmed experimentally. Along the way, we comment on the ‘parameter estimation problem’ and conclude with an appeal for a greater role for mathematical models in molecular cell biology.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
                  
                  
                    <!-- Portfolio Grid Section -->



<li class="timeline-inverted">

  <div class="timeline-image">
      <img class="img-circle img-responsive" src="https://novakgroupoxford.github.io/bicycle-hugo//img/publication/pk_febsletters_2015.png" alt="">
  </div>
  <div class="timeline-panel">
    <div class="timeline-heading">
      <h6 class="subheading">Thursday, Feb 12, 2015</h6>
      <h5 class="subheading">FEBS Letters</h5>
      <a href="/bicycle-hugo/publication/pk_febsletters_2015/"><h4>Model scenarios for switch-like mitotic transitions</h4></a>
      <h5 class="subheading">
        Vinod PK, Bela Novak
      </h5>
      <ul class="collapsible" data-collapsible="expandable">
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-doi"></i>
              <a href="https://dx.doi.org/10.1016/j.febslet.2015.02.007" target="_blank" title="Visit publisher website">https://dx.doi.org/10.1016/j.febslet.2015.02.007</a></div>
          </div>
        </li>
        <li>
          <div>
            <div class="collapsible-header">
              <i class="ai ai-pubmed"></i>
              <a href="https://www.ncbi.nlm.nih.gov/pubmed/?term=10.1016%2fj.febslet.2015.02.007" target="_blank" title="Search article in pubmed">View on pubmed</a>
            </div>
          </div>
        </li>
        
        
        <li>
          <div class="collapsible-header"><i class="fa fa-file-text"></i>Abstract</div>
          <div class="collapsible-body"><span>To facilitate rapid accumulation of Cdk1-phosphorylated substrate proteins, the Cdk1 counter-acting phosphatase, PP2A-B55 is inhibited during M phase by stoichiometric inhibitors (ENSA and Arpp19). These inhibitors are activated when phosphorylated by Cdk1-activated Greatwall-kinase. Recent experiments show that ENSA is dephosphorylated and inactivated by the PP2A-B55 itself, and acts as an unfair substrate inhibiting PP2A-B55 activity towards other Cdk1 substrates. Mathematical modelling shows that this mutual antagonism between the phosphatase and its inhibitor is insufficient to explain the switch-like characteristics of mitotic entry and exit. We show that the feedback regulation of Greatwall activating kinase and/or inactivating phosphatase can explain the abruptness of these cell cycle transitions.</span></div>
        </li>
      </ul>

    </div>
  </div>
</li>

                  
                
              </ul>
            </div>
          </div>
        </div>
      </div>
    </div>

    <!-- jQuery -->
<script src="https://novakgroupoxford.github.io/bicycle-hugo/js/jquery-v3.3.1/jquery.min.js"></script>

<!-- Bootstrap Core JavaScript -->
<script src="https://novakgroupoxford.github.io/bicycle-hugo/js/bootstrap-v3.3.7/bootstrap.js"></script>

<!-- Plugin JavaScript -->
<script src="https://novakgroupoxford.github.io/bicycle-hugo/js/classie-v1.0.1.js"></script>
<script src="https://novakgroupoxford.github.io/bicycle-hugo/js/cbpAnimatedHeader-v1.0.0.js"></script>

<!-- Contact Form JavaScript -->
<script src="https://novakgroupoxford.github.io/bicycle-hugo/js/jqBootstrapValidation-v1.3.6-dirty.js"></script>
<script src="https://novakgroupoxford.github.io/bicycle-hugo/js/contact_me.js"></script>

<!-- Custom Theme JavaScript -->
<script src="https://novakgroupoxford.github.io/bicycle-hugo/js/agency.js"></script>

<script type="text/javascript" src="https://code.jquery.com/jquery-2.1.1.min.js"></script>
<script type="text/javascript" src="https://novakgroupoxford.github.io/bicycle-hugo/js/materialize.min.js"></script>







  </body>
</html>
