+++
categories = []
tags = []
date = "2017-01-12T14:47:12+01:00"
title = "Our Research"

+++
# Our Research

During its growth and division cycle the cell replicates all of its components and distributes them to create two daughter cells [1]. It is essential that the underlying processes, DNA replication (S phase), mitosis (M phase) and cell division (CD), take place in a defined sequence with the correct temporal separation: G1 → S → G2 → M → CD → G1 → . Progression through this sequence is characterized by specific transitions at boundaries between sequential stages of the cell cycle. At the **Restriction point** in G1 phase the cell decides whether to proliferate or quiesce (G0). This is followed later by the abrupt **G1/S transition** into S phase (DNA replication). Post-replicative cells in G2 phase enter into mitosis (M phase) at the **G2/M transition**. The decision to segregate sister-chromatids once all have become aligned pairwise on the mitotic spindle takes place at the end of prometaphase and triggers the **meta/anaphase transition and cytokinesis**. Anaphase is followed by the exit from mitosis back into G1 (mitotic exit). Progression through the cell cycle in eukaryotes is controlled by a conserved protein interaction network composed of protein kinases and phosphatases with stoichiometric inhibitors and activators, transcription factors, ubiquitin-conjugating enzymes and the proteasome. The molecular interactions among these cell cycle regulators create a complicated **dynamical system** that oscillates once per cell cycle. In our proposal we focus on a systems-level description of the major cell cycle transitions in the mammalian cell cycle. Our **hypothesis-driven** research is based on our theoretical investigations of generic dynamic characteristics in eukaryotic cell cycle transitions.

{{< figure src="https://novakgroupoxford.github.io/bicycle-hugo/img/activator-inhibitor.png" title="Fig.1: Generic network motif of cell cycle transitions" >}}
*The three components are embedded in a negative feedback loop. The antagonism between Activator<sub>i</sub> and Inhibitor<sub>i</sub> creates a positive circuit.*

Progression through each of the cell cycle transitions is controlled by a balance between **inhibitors** and **activators** (Fig.1). Before each cell cycle transition, an inhibitor overwhelms the activator responsible for the transition (refered here as **Inhibitor<sub>i</sub>** and **Activator<sub>i</sub>**). During the transition Inhibitor<sub>i</sub> gets down-regulated by the activator of the previous cell cycle stage (Activator<sub>i-1</sub>) which leads to activation of Activator<sub>i</sub>. However after the transition, Activator<sub>i</sub> inhibits the activator characteristic for the earlier cell cyle stage (Activator<sub>i-1</sub>), which creates a **negative feedback loop** in the network [2]. The **irreversibility** of cell cycle transitions requires that Inhibitor<sub>i</sub> is kept inactive in the post-transition state despite the disappearance of Activator<sub>i-1</sub>. We argue that the irreversibility of the cell cycle transition is achieved by an **antagonistic interaction** between Activator<sub>i</sub> and its cognate Inhibitor<sub>i</sub>. This **double-negative feedback** makes the post-transition state and Activator<sub>i</sub> self-maintaining by creating a **bistable switch** (Fig.2). The irreversible switch creates a **‘point of no return’** in the cell cycle transition, which can be experimentally tested by the following protocol. In a partly synchronised cell population undergoing the transition, abrupt inactivation of Activatori-1 redirects cells back to the bistable region close to the y-axis. The future cellular fate will be dependent upon whether cells end up above or below the unstable saddle point (Fig.3). Cells early in the transition will return to the pre-transition cell cycle state (cell ) with low Activator<sub>i</sub> and high Inhibitor<sub>i</sub> levels. In contrast, cells later in the transition will manage to reach the post-transition (i) state (cell ) where the levels of Activator<sub>i</sub> and Inhibitor<sub>i</sub> are the opposite.

{{< figure src="https://novakgroupoxford.github.io/bicycle-hugo/img/phaseplane.png" title="Fig.2: Two phaseplane portraits of the generic cell cycle switch" >}}
*At small Activator<sub>i-1</sub> levels the network has two alternative steady states (solid lines) separated by unstable saddle points (dashed lines). The trajectories are shown by dotted curves.*

The sequential connection of these switch-like network motifs not only accounts for oscillations of activators and inhibitors during cell cycle progression, but also provides **‘checkpoint’** control over the cell cycle transitions. By monitoring defective states of chromosomes (damaged, not fully replicated, or improperly aligned on the mitotic spindle), intracellular surveillance mechanisms keep Activator<sub>i-1</sub> at a subthreshold level and thereby block the transition. When the defect is repaired, the bistable switch is released to drive the transition as shown (Fig.2).
