+++
title = "chris_bakal"
date = "2013-11-30"
name = "Chris Bakal"
roles = ["Principal Investigator"]
groups = ["Bakal"]
universities = ["ICR London"]
img = "chris_bakal.jpg"
[Social]
  twitter = ""
  googlescholar = ""
  researchgate = ""
  linkedin = ""
  github = ""
  website = ""
+++
Chris Bakalâ€™s contributions to biology have been to use imaging and computational methods to describe the signalling networks regulating cell shape. Specifically, the Bakal laboratory has developed the use of Quantitative Morphological Signatures (QMSs) to describe the contribution of individual genes to the regulation of single-cell shape (Bakal et al., Science 2007), automated online phenotype discovery algorithms that search through databases containing millions of images to identify novel cell shapes (Yin et al., BMC Bioinformatics 2008), algorithms to determine hits from RNAi screens based on the known physical structure of signalling networks (Kraplow et al., Nature Methods 2009), combinatorial RNAi methods to determine epistatic relationships in metazoan systems (Arias-Garcia et al., Molecular Biosystems 2012), methods to describe networks through integration of phosphoproteomic and functional genomic data (Bakal et al., Science 2008), and algorithms to model hierarchical signalling relationships from QMSs derived from combinatorial RNAi (Nir et al., Genome Research 2010). Recently the Bakal laboratory devised methods to quantify the morphological heterogeneity of populations in the context of a high-throughput genetic screen, and identified a signalling network that regulates the switch-like changes in cell shape made by melanoma cells during the metastatic process (Yin et al., Nature Cell Biology 2013). The Bakal laboratory will focus on G1/S transition and will be responsible for the live cell imaging facilities as well.
